lightly_studio/tests/api/routes/api/test_video.py

```diff
@@ -1,26 +1,15 @@
+from uuid import UUID
+
 from fastapi.testclient import TestClient
 from sqlmodel import Session
 
 from lightly_studio.api.routes.api.status import HTTP_STATUS_OK
-from lightly_studio.models.dataset import SampleType
-from tests.helpers_resolvers import (
-    create_dataset,
-)
-from tests.resolvers.video_resolver.helpers import VideoStub, create_videos
+from tests.resolvers.video_resolver.helpers import create_videos_to_fake_dataset
 
 
 def test_get_all_videos(test_client: TestClient, db_session: Session) -> None:
-    dataset = create_dataset(session=db_session, sample_type=SampleType.VIDEO)
-    dataset_id = dataset.dataset_id
-
-    create_videos(
-        session=db_session,
-        dataset_id=dataset_id,
-        videos=[
-            VideoStub(path="/path/to/sample1.mp4"),
-            VideoStub(path="/path/to/sample2.mp4"),
-        ],
-    )
+    videos = create_videos_to_fake_dataset(db_session=db_session)
```

Comment:
Please do not create a helper function. Instead, copy the test setup code.

As explained in our testing tech talk and also in the post by Lukas, we want tests to be DAMP and not DRY. The tests should be easily explainable, therefore it is better to keep the test setup visible in the test functions. DRY (don't repeat yourself) applies for the regular non-test source code.